arnab_bhadra/arnabbhadra_PostgresAssignment/src/Services/ClassModel.ts

```diff
@@ -0,0 +1,6 @@
+import { client } from "../Database/Client";
+
+export const insertClassScheduleInfoIntoDB = async (classScheduleinfo: any) => {
```

Comment:
We don't need to add **intoDB** suffix . It is upto service later to decide where the data will be stored. Naming it like this will tightly couple things :)